src/Dialogs/BulkRenamer/RenamerDialog.vala

```diff
@@ -197,7 +204,7 @@ public class Files.RenamerDialog : Gtk.Dialog {
                     replace_entry_revealer.reveal_child = false;
                     break;
                 case RenameBase.REPLACE:
-                    basename_entry.placeholder_text = _("Enter text of original to be replaced");
+                    basename_entry.placeholder_text = _("Text to be replaced");
```

Comment:
These should really be labels outside of the entry. Placeholder text should be an example of valid input, it shouldn't be a description for the entry. But less verbose is at least visible